### ISSUE TITLE
Add Blauth grading thumb hypoplasia

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10459]
**Preferred term label:**
Thumb hypoplasia grade 1

**Synonyms**
Hypoplasia of the thumb grade I, 

**Definition (free text, please give PubMed ID)**
Blauth classification. Minor hypoplasia, all musculoskeletal and neurovascular components of the digit are present, just small in size

**Parent term (use hpo.jax.org/app)**
Short thumb HP:0009778

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
See the disease associations from Short thumb HP:0009778

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000938
- label: Thumb hypoplasia grade 1
- parents: [Short thumb (HP:0009778)](https://hpo.jax.org/app/browse/term/HP:0009778)
- pmid: PMID:24362256
- def: Minor hypoplasia (developmental undergrowth) of the thumb, whereby all musculoskeletal and neurovascular components of the digit are present, just small in size (Blauth classification).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10459
- synonyms:
